index.html

```diff
@@ -164,7 +159,6 @@ <h3 class="cocogoose text-dark text-center h4 fw-bold pt-5">Featured Instructors
         </div>
       </section>
 
-      <!-- partners -->
       <section id="partner" class="partner-bg toggle-1">
         <div class="container-md p-3">
           <h3 class="text-partner text-center h3">Partners</h3>
```

Comment:
- [ ] Please increase the size of the partner's image and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/158642005-5043e1f1-06da-4e00-8d4f-1c1e02cd403f.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/158642049-22720669-4832-4a45-9a80-f5d45f44b124.png)